### PR TITLE
Fix issue #280

### PR DESCRIPTION
**Pull request by subhero24**

### CODE REVIEW

File: test/issue-280.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const FindMyWay = require('../')
+
+test('Wildcard route match when regexp routes', (t) => {
```
Comment (ivan-tymoshenko) [inline test/issue-280.test.js:7]:
```suggestion
test('Wildcard route match when regexp route fails', (t) => {
```

**Review by ivan-tymoshenko — APPROVED** [review]
LGTM

**Comment by ivan-tymoshenko** [comment]
Can you add one more test where after regexp parametric note it matches parametric route?
```js
  findMyWay.on('GET', '/:a(a)', () => {})
  findMyWay.on('GET', '/:a/static', () => {})
  findMyWay.on('GET', '/*', () => {})
  
  findMyWay.find('GET', '/b/static', {})
```

File: test/issue-280.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const FindMyWay = require('../')
+
+test('Wildcard route match when regexp route fails', (t) => {
+  t.plan(1)
+  const findMyWay = FindMyWay()
+
+  findMyWay.on('GET', '/:a(a)', () => {})
+  findMyWay.on('GET', '/*', () => {})
+
+  t.same(findMyWay.find('GET', '/b', {}).params, { '*': '/b' })
```
Comment (ivan-tymoshenko) [inline test/issue-280.test.js:14]:
```suggestion
  t.same(findMyWay.find('GET', '/b', {}).params, { '*': 'b' })
```

**Comment by mcollina** [comment]
@ivan-tymoshenko could you take a look?

**Comment by ivan-tymoshenko** [comment]
still LGTM

**Pull request merged**